Longest Word in Dictionary.py

```diff
@@ -0,0 +1,41 @@
+# Time Complexity : O(N), where N is the length of word searched / inserted
+# Space Complexity : O(N), For storing every character in a word, we have a TrieNode that uses an array of fixed size of 26
+# Did this code successfully run on Leetcode : YES
+
```

Comment:
Space complexity is  nk. where n is number of words in list to be inserted in trie and k is length of longest word in dictionart